test/SlowTests/SlowTests/MailingList/Jalchr.cs

```diff
@@ -1,212 +0,0 @@
-// -----------------------------------------------------------------------
-//  <copyright file="Jalchr.cs" company="Hibernating Rhinos LTD">
-//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
-//  </copyright>
-// -----------------------------------------------------------------------
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using FastTests;
-using Newtonsoft.Json;
-using Raven.Client.Documents.Indexes;
-using Raven.Client.Documents.Linq;
-using Raven.Client.Documents.Transformers;
-using Xunit;
-
-namespace SlowTests.SlowTests.MailingList
-{
-    public class Jalchr : RavenTestBase
-    {
-        [Fact]
-        public void CanGetIdWithCorrectCase()
-        {
-            using (var store = GetDocumentStore())
-            {
-                store.Conventions.CustomizeJsonSerializer = serializer =>
-                               serializer.ReferenceLoopHandling = ReferenceLoopHandling.Ignore;
```

Comment:
Restore candidate? due to `serializer.ReferenceLoopHandling = ReferenceLoopHandling.Ignore;`